tox.ini

```diff
@@ -50,6 +50,7 @@ deps =
 
 [testenv:py27]
 deps =
+    pathlib
```

Comment:
pls add to ``ci/requirements-2.7.run``, this is SOLELY for local testing and doesn't hit travis